### ISSUE TITLE
Google Landmark Federated split is missing files

### ISSUE THREAD

**Issue by marcociccone** [google-research/google-research#946]
Hi!
I've started working with the gldv2 dataset for FL using the split provided [here](https://github.com/google-research/google-research/tree/master/federated_vision_datasets).
In particular, I'm using the [dataset loader](https://www.tensorflow.org/federated/api_docs/python/tff/simulation/datasets/gldv2/load_data) from tensorflow federated.

I've noticed that roughly a 10% of images are missing: [this](https://github.com/tensorflow/federated/blob/da2ca219e75e3317c43e10a83d07b635aad9cfd2/tensorflow_federated/python/simulation/datasets/gldv2.py#L108) exception is raised when creating the clients tfrecords.
It looks that some filenames in the federated splits are not matching filenames from the original dataset. 

I've collected the missing files in a [json file](https://drive.google.com/file/d/1HbMKi8LrWCwldhl7H4DuZmEJcLBSavgq/view?usp=sharing) that you can check (list of dict - one element per client - with fields `user_id`, `total`, `found`, `missing`, `missing_files`)

@hang-qi I'm keeping bugging you sorry :)

**Comment by hang-qi** [comment]
@marcociccone Can you confirm if the correct full zip package was downloaded? You may clear the [cache dir][cache], which will allow the script to redownload the package.

https://github.com/tensorflow/federated/blob/v0.19.0/tensorflow_federated/python/simulation/datasets/gldv2.py#L36-L38

[cache]: https://github.com/tensorflow/federated/blob/v0.19.0/tensorflow_federated/python/simulation/datasets/gldv2.py#L220

**Comment by marcociccone** [comment]
yes, I'm pretty sure that the split file is correct. I've used that script and debugged it heavily.
Cache is fine. I've built the dataset from scratch to check this.
